app/models/recipe.rb

```diff
@@ -1,3 +1,4 @@
 class Recipe < ApplicationRecord
+   has_many :ingredients 
    validates :name, :complexity, :genre, presence: true
 end
```

Comment:
Remember that when adding a `has_many` relationship, it's because this table's primary key is a foreign key on the child table. I do not recall there being a foreign key for a recipe on the ingredients table. Furthermore, since this IC was built around a many-to-many relationship, the only way that a recipe could have many ingredients is through a joins table.

The tests for the association were already created, all you needed to do was add those associations in their respective models to get all the tests to pass.